hadoop-hdds/docs/content/feature/OM-HA.md

```diff
@@ -27,15 +27,15 @@ Ozone has two leader nodes (*Ozone Manager* for key space management and *Storag
 
 To avoid any single point of failure the leader nodes also should have a HA setup.
 
- 1. HA of Ozone Manager is implemented with the help of RAFT (Apache Ratis)
- 2. HA of Storage Container Manager is [under implementation]({{< ref "scmha.md">}})
+Both Ozone Manager and Storage Container Manager supports HA. In this mode the internal state is replicated via RAFT (with Apache Ratis) 
+
+This document explain the HA setup of Ozone Manager (OM) HA, please check [this page[({{< ref "SCM-HA" >}})].  While they can be setup for HA independently, a reliable, full HA setup requires enabling HA for both services.
```

Comment:
Out of curiosity: 

Is `[this page[({{< ref "SCM-HA" >}})]` markdown style? Is there a way to verify this is a valid link to SCM-HA doc?

Reply:
No, it's not valid.  It renders as:

```
please check [this page[(/feature/scm-ha.html)].
```

```suggestion
This document explain the HA setup of Ozone Manager (OM) HA, please check [this page]({{< ref "SCM-HA" >}}) for SCM HA.  While they can be setup for HA independently, a reliable, full HA setup requires enabling HA for both services.
```

Steps to verify:

```
mvn -pl :hadoop-hdds-docs clean package
open hadoop-hdds/docs/target/classes/docs/feature/om-ha.html
```